### PR TITLE
F#213 add dc motor driver

### LINKED ISSUE #213: Add new DC motor driver
Talk with @Rocketmagnet for the protocol

### PR DESCRIPTION
**Pull request by VahidAminZ**
fixes #213

### CODE REVIEW

File: sr_ronex_msgs/msg/DCMotorState.msg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Header header
+
+bool[] digital
+uint16[] analogue
+bool[] input_mode
+MotorPacketStatus[] motor_status
```
Comment (kirstyellis) [inline sr_ronex_msgs/msg/DCMotorState.msg]: newline required at end of these next 3 files

File: sr_ronex_external_protocol/include/sr_ronex_external_protocol/Ronex_Protocol_0x02000009_DC_Motor_Small_00.h [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+#define PROTOCOL_TYPE   EC_QUEUED                                           //  Synchronous communication
+#define COMMAND_ADDRESS 0x1000                                              //!< ET1200 address containing the Command Structure
+#define STATUS_ADDRESS  (COMMAND_ADDRESS+sizeof(RONEX_COMMAND_02000009) *4) //!< ET1200 address containing the Status  Structure
+
+#define RONEX_COMMAND_STRUCT        RONEX_COMMAND_02000009                  //!< Required for et1200_interface.h to be generic
+#define RONEX_STATUS_STRUCT         RONEX_STATUS_02000009                   //!< Required for et1200_interface.h to be generic
+
+
+
+#endif
```
Comment (kirstyellis) [inline sr_ronex_external_protocol/include/sr_ronex_external_protocol/Ronex_Protocol_0x02000009_DC_Motor_Small_00.h]: end of line error

File: sr_ronex_controllers/src/DC_motor_small_passthrough_controller.cpp [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+{
+  for (size_t index = 0; index < dc_motor_small_->command_.motor_packet_command_.size(); ++index)
+  {
+    ROS_INFO_STREAM("Stopping DC motor");
+    dc_motor_small_->command_.motor_packet_command_[index].flags = 0;
+    dc_motor_small_->command_.motor_packet_command_[index].on_time = 0;
+    dc_motor_small_->command_.motor_packet_command_[index].period = 0;
+  }
+}
+}  // namespace ronex
```
Comment (kirstyellis) [inline sr_ronex_controllers/src/DC_motor_small_passthrough_controller.cpp:126]: no emacs weenies comment

Comment (ugocupcic) [inline sr_ronex_controllers/src/DC_motor_small_passthrough_controller.cpp:126]: :+1:

**Pull request merged**